core/src/components/picker-column/picker-column.tsx

```diff
@@ -28,8 +28,8 @@ export class PickerColumnCmp implements ComponentInterface {
   private y = 0;
   private optsEl?: HTMLElement;
   private gesture?: Gesture;
-  private rafId: any;
-  private tmrId: any;
+  private rafId?: ReturnType<typeof requestAnimationFrame>;
```

Comment:
`requestAnimationFrame` returns a `number` type. We should use the primitive type.

Reply:
I like to use the more explicit `ReturnType` call for RAFs since it's easier to read; a `number` type doesn't tell as much about how the variable is meant to be used. I can use the primitive if you feel strongly about it though.

Reply:
Works for me!